endpoints.md

```diff
@@ -37,6 +37,7 @@
 * [/api/workflow/pooltasks](pooltasks.md)
 * [/api/workflow/claimedtasks](claimedtasks.md)
 * [/api/tools/feedbacks](feedbacks.md)
+* [/api/eperson/profiles](profiles.md)
```

Comment:
Tiny thing, we should consider moving this up next to `/api/eperson/epersons`, since it's somewhat related to that endpoint.